phpcs.xml

```diff
@@ -77,7 +77,7 @@
 	</rule>
 
 	<!-- Include sniffs for PHP cross-version compatibility. -->
-	<config name="testVersion" value="5.4-99.0"/>
+	<config name="testVersion" value="5.4-"/>
 	<rule ref="PHPCompatibility">
```

Comment:
You need to use the replacement ruleset for it to take effect:
```suggestion
	<rule ref="PHPCompatibilityWP">
```

Reply:
D'oh, I thought I did that.